en/django_forms/README.md

```diff
@@ -44,36 +44,48 @@ So once again we will create a link to the page, a URL, a view and a template.
 
 ## Link to a page with the form
 
-It's time to open `blog/templates/blog/base.html` in the code editor. In the `div` named `page-header`, we will add a link:
+Before we add the link, we need some icons to use as buttons for the link. For this tutorial, download [file-earmark-plus.svg](https://raw.githubusercontent.com/twbs/icons/main/icons/file-earmark-plus.svg) and save it in the folder `blog/templates/blog/icons/`
+
+> Note: To download the svg image, click on the link and save the file using the keyboard shortcut `Ctrl + S`(Windows/Linux) or `Cmd + S`(MacOS).
```

Comment:
Please write the abbreviation with capital letters:
> SVG

where it occurs in prose. (Don't change it in file names, as they are case sensitive on Linux.)

Reply:
Resolved in https://github.com/DjangoGirls/tutorial/pull/1681/commits/4d6ffbc4920caa307ba7a8e13619ce8b8237f4fc